modules/aws/vpc/README.md

```diff
@@ -78,19 +84,25 @@ repository you can specify a version by appending the below source reference wit
 [here](https://developer.hashicorp.com/terraform/language/modules/sources#modules-in-package-sub-directories))
 
 ```hcl
-module "vpc_subnet" {
-  source       = "github.com/answerdigital/terraform-modules//modules/aws/vpc?ref=v1.0.0"
-  owner        = "joe_blogs"
-  project_name = "example_project_name"
-}
-
-module "vpc_subnet" {
-  source               = "github.com/answerdigital/terraform-modules//modules/aws/vpc?ref=v1.0.0"
+module "vpc_subnet" "mandatory" {
+  source               = "github.com/answerdigital/terraform-modules/modules/aws/vpc?ref=vx.x.x"
   owner                = "joe_blogs"
   project_name         = "example_project_name"
-  azs                  = ["eu-west-1", "eu-west-3"]
-  num_public_subnets   = 1
-  num_private_subnets  = 2
   enable_vpc_flow_logs = true
 }
+
+module "vpc_subnet" "all" {
+  source                     = "github.com/answerdigital/terraform-modules/modules/aws/vpc?ref=vx.x.x"
+  owner                      = "joe_blogs"
+  project_name               = "example_project_name"
+  azs                        = ["eu-west-1", "eu-west-3"]
+  num_public_subnets         = 1
+  num_private_subnets        = 2
+  enable_vpc_flow_logs       = true
+  vpc_flow_logs_traffic_type = "ALL"
+  enable_dns_support         = true
+  enable_dns_hostnames       = true
+  ig_cidr                    = "0.0.0.0/24"
```

Comment:
I don't think this is right? Its default is "0.0.0.0/0" to allow all traffic. Therefore I wouldn't specify this in the example.